packages/components/src/components/hds/app-frame/parts/main.hbs

```diff
@@ -2,6 +2,6 @@
   Copyright (c) HashiCorp, Inc.
   SPDX-License-Identifier: MPL-2.0
 }}
-<main class="hds-app-frame__main" ...attributes>
+<main class="hds-app-frame__main" id="hds-main" ...attributes>
```

Comment:
noting that, if they want to give a different `id` to the `<main>` element they will be able to do so using the `id` attribute (as before) and match it with a `@a11yRefocusSkipTo`

Reply:
WIth the `...attributes`, when you pass an attribute that is set before it (like if they set a custom id) does the attribute from the splat override the initial one ("hds-main" in this example)?

I feel like we usually have a getter in situations like this where there is a default attribute for a value that a consumer can change. Example: https://github.com/hashicorp/design-system/blob/a9473e0333fde6d42534f45a41017691b27bc913/packages/components/src/components/hds/breadcrumb/truncation.ts#L24

Reply:
I'll add a test as @alex-ju suggested below for a custom id (I tested locally manually but didn't write a test for it)

I'm not sure if we need a getter in this case? @alex-ju what do you think?

Reply:
> WIth the ...attributes, when you pass an attribute that is set before it (like if they set a custom id) does the attribute from the splat override the initial one ("hds-main" in this example)?

yes, when written as such, the splattribute overrides the pre-set attribute

Reply:
> I'll add a test as @alex-ju suggested below for a custom id (I tested locally manually but didn't write a test for it)
> I'm not sure if we need a getter in this case? @alex-ju what do you think?

no getter needed, it works as expected as is

Reply:
Added tests for the AppFrame::Main id (default & custom values)